Server/Game/tasks.py

```diff
@@ -4,122 +4,123 @@
 
-    # Randomly choose a second player from the remaining players on the queue
-    index = random.randint(1, len(queue) - 1)
-    second_player = queue[index]
+        # Randomly choose a second player from the remaining players on the queue
+        index = random.randint(1, len(queue) - 1)
+        second_player = queue[index]
 
-    logger.info("Found a match: " + str(first_player.user.username) + " vs. " + str(second_player.user.username))
+        logger.info("Found a match: " + str(first_player.user.username) + " vs. " + str(second_player.user.username))
 
-    # Determine the number of times these 2 players have played against each other
-    count = 0
-    for first_player_game_user in Game_User.objects.filter(user=first_player.user):
-        game_played = first_player_game_user.game
+        # Determine the number of times these 2 players have played against each other
+        count = 0
+        for first_player_game_user in Game_User.objects.filter(user=first_player.user):
+            game_played = first_player_game_user.game
 
-        if not game_played:
-            continue
+            if not game_played:
+                continue
 
-        is_second_player_in_game = Game_User.objects.filter(user=second_player.user, game=game_played).first()
+            is_second_player_in_game = Game_User.objects.filter(user=second_player.user, game=game_played).first()
 
-        if is_second_player_in_game:
-            count += 1
+            if is_second_player_in_game:
+                count += 1
 
-    logger.info("Users have played each other " + str(count) + " times.")
+        logger.info("Users have played each other " + str(count) + " times.")
 
-    # Randomly choose which player will start first
-    player_turn_index = random.randint(1, 2)
-    player_turn = None
-    if player_turn_index == 1:
-        logger.info(str(first_player.user.username) + " will take the first turn")
-        player_turn = first_player.user
-    elif player_turn_index == 2:
-        logger.info(str(second_player.user.username) + " will take the first turn")
-        player_turn = second_player.user
+        # Randomly choose which player will start first
+        player_turn_index = random.randint(1, 2)
+        player_turn = None
+        if player_turn_index == 1:
+            logger.info(str(first_player.user.username) + " will take the first turn")
+            player_turn = first_player.user
+        elif player_turn_index == 2:
+            logger.info(str(second_player.user.username) + " will take the first turn")
+            player_turn = second_player.user
 
-    if not mock_maps:
         maps = Map.objects
-    else:
-        maps = mock_maps
 
-    # Randomly choose a map for the game
-    index = random.randint(0, maps.count() - 1)
-    game_map = maps.filter()[index]
+        # Randomly choose a map for the game
+        index = random.randint(0, maps.count() - 1)
+        game_map = maps.filter()[index]
```

Comment:
Actually, I did it like that because I wanted to also use .count() on the objects.  If maps contains the filter, the next line that does maps.count() won't work on the filter() list.